index.html

```diff
@@ -38,5 +38,94 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-3">
+            <div class="card-3-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-4">
+            <div class="card-4-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="ams">
+            <h2>About Myself</h2>
+            <p>
+               Hello I'm a software developer! I can help you build a product, feature or website
+               Look through some of my work and experiance! If you like what you see and have a project you need coded,
+               don't hestiate to contact me.
+            </p>
+            <h3 class="letconnect_text"><a href="#">LET'S CONNECT</a></h3>
+            <div class="letconnect_icons">
+               <ul>
+                  <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+                  <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+               </ul>
+            </div>
+            <button>Get my resume</button>
+         <div class="About-languages">
+               <ul>
+                  <li class="about-lang-menu"><a href="#"><button><h3>Languages</h3><img src="Images\about_down_arrow.svg" alt="" width="14px" height="8px"></button></a></li>
```

Comment:
- [ ] Please ensure that this text `Languages` is also styled properly to match that of the Figma design. Currently, it looks smaller and has a different `font family` to what was used in the Figma design. Please ensure that you apply the right styles to the texts as used in the Original Design as I mentioned before :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/227709458-545b60d9-1fb9-459d-9ddd-89f5c003d6e9.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/227709473-540400b4-a852-4205-b51d-1cdc8bdfe403.png)